scripts/bump-version.sh

```diff
@@ -21,9 +21,6 @@ check_version(){
     echo ""
 }
 
-echo "* Bumping version in README.adoc"
```

Comment:
Why you removed these three line ? Are they not useful atm ?

Reply:
Yes, they are not useful anymore, as the version number is not mentioned anywhere in the README. anymore

Reply:
we don't have any version information in the readme right now

Reply:
Thanks for the info